bundles/org.openhab.binding.shieldtv/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="shieldtv" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>ShieldTV Binding</name>
+	<description>This is the binding for ShieldTV.</description>
+
+</binding:binding>
```

Comment:
Please convert to the new `addon.xml`.

Reply:
Will do. I'll push it with some other things this week.